src/components/Components.tsx

```diff
@@ -141,49 +135,50 @@ export const MaterialInput: React.FunctionComponent<IMaterialInput> = props => {
       // notchedOutline: classes.notchedOutline,
     },
     readOnly: props.readOnly,
+    spellCheck: false,
```

Comment:
Why we disable browser's spellcheck?

Reply:
Since it can be often the case that step, pipeline and experiment names are not proper english words, it can be annoying to alway have the red underline.